server/server.js

```diff
@@ -462,6 +473,7 @@ app.patch(`${apiBaseUrl}/employer-sites/:id`,
     await validate(EditSiteSchema, req.body);
     const user = req.hcapUserInfo;
     try {
+      req.body.coords = await getPointsFromPostalCodes(req.body.postalCode);
```

Comment:
What does assigning to the request do here?

Reply:
This is actually obsolete code, good catch. The req.body is passed to the updateSite function with the coords that are derived from the postal code, but we're now updating the coords via database trigger.